pkgs/shelf_router/lib/src/route.dart

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 import 'package:meta/meta.dart' show sealed;
-import 'package:shelf_router/src/router.dart';
+import 'router.dart';
```

Comment:
organize imports?

Reply:
prefer relative imports...

Reply:
For sure, but this changed the import from a package to a file import, and the 'organize import' command would insert a blank line between these two imports now :)

Reply:
ah! Hrm...yup